hcl/ast/attribute.go

```diff
@@ -33,7 +34,7 @@ type Attributes map[string]Attribute
 func NewAttribute(origin string, val *hclsyntax.Attribute) Attribute {
```

Comment:
hmm true, at first I didn't wanted to change the code too much, but for the testing it will be easier and it makes more sense since we want to work as much as possible with the hcl types.